### PR TITLE
Cleanup and rename

### PR DESCRIPTION
**Pull request by sertonix**
This pull request does the following:
- remove unused `.md` files
- updated `CONTRIBUTING.md`
- removed badges from readme
- change repository url
- rename/remove `Atom` references

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
 # Package Generator package
-[![OS X Build Status](https://travis-ci.org/atom/package-generator.svg?branch=master)](https://travis-ci.org/atom/package-generator)
-[![Windows Build Status](https://ci.appveyor.com/api/projects/status/7t1i4hdmljhigp9u/branch/master?svg=true)](https://ci.appveyor.com/project/Atom/package-generator/branch/master) [![Dependency Status](https://david-dm.org/atom/package-generator.svg)](https://david-dm.org/atom/package-generator)
-
 
 Generates and opens a new sample package, language, or syntax theme in Atom.
```
Comment (confused-Techie) [inline README.md]:
```suggestion
Generates and opens a new sample package, language, or syntax theme in Pulsar.
```

**Review by confused-Techie — CHANGES_REQUESTED** [review]
This overall looks fantastic, but added one change, since we might as well be thorough on here. Thanks for contributing!

**Review by confused-Techie — APPROVED** [review]
Fantastic, thanks for getting on those changes. This looks rad!

**Comment by sertonix** [comment]
Close in favor of #2

**Comment by confused-Techie** [comment]
@Sertonix we don't have to close this one if you don't want. If the PR on the main editor was accepted I was going to merge this one, then right after #2, if you don't mind we can leave it closed, but didn't want to discount the contribution you've made to this repo

**Comment by sertonix** [comment]
It is mentioned in the pull request so this is not needed

**Pull request closed without merging**